app/services/imports/lettings_logs_field_import_service.rb

```diff
@@ -8,13 +8,32 @@ def update_field(field, folder)
         import_from(folder, :update_major_repairs)
       when "lettings_allocation"
         import_from(folder, :update_lettings_allocation)
+      when "offered"
```

Comment:
Please could you add a spec for this change? See: `spec/services/imports/lettings_logs_field_import_service_spec.rb`

Reply:
Have done so - although I have missed out the "value we're trying to re-import is invalid" case because the set up effort was becoming a bit more than it seemed worth.